src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stopped its work");
+    }
```

Comment:
`Override` annotation is documented as not compulsory, but it is very recommended to add it